interpreter/interpreter_test.go

```diff
@@ -383,6 +385,19 @@ func TestEval(t *testing.T) {
 				values.NewBool(false),
 			},
 		},
+		{
+			name:    "invalid function parameter",
+			query:   `from(bucket: "telegraf") |> window(every: 0s)`,
+			wantErr: `error calling function "window" @\d+:\d+-\d+:\d+: every parameter must be nonzero`,
```

Comment:
I would have thought that a file name would appear in this message somewhere

Reply:
The parser isn't invoked with a file name so no filename gets included.

Reply:
Ok, maybe we need an issue to decide on a format for errors with source location. I like something like

```
(input)@line:col-line:col: error calling function "window": "every" parameter must be nonzero
```
Where we have a reasonable default like `(input)` when on filename was supplied.

But that could be done outside of this PR.